js/w3c/style.js

```diff
@@ -73,15 +73,16 @@ define(
             case "LC-NOTE":
               styleFile += "WD";
               break;
+            case "WG-NOTE":
             case "FPWD-NOTE":
-              styleFile += "WG-NOTE";
+              styleFile += "WG-NOTE.css";
```

Comment:
Why did you add .css to this?  I thought they were using conneg to get the file we want.